src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt

```diff
@@ -119,6 +119,7 @@ private fun mapOuterMetaDataToColumnSql(metadataOption: MetadataOption): String
         // TODO should be just last_write with comma prefix after empty rows are eliminated https://jira.openlattice.com/browse/LATTICE-2254
         MetadataOption.LAST_WRITE -> ",max(${LAST_WRITE.name}) AS ${mapMetaDataToColumnName(metadataOption)}"
         MetadataOption.ENTITY_KEY_IDS -> ",array_agg(${ORIGIN_ID.name}) as ${ENTITY_KEY_IDS_COL.name}"
+        MetadataOption.VERSION -> ",max(${VERSION.name}) As ${mapMetaDataToColumnName(metadataOption)}"
```

Comment:
Just a minor thing, do we need this metadata option anymore? What was the purpose of it?

Reply:
correct, we don't.